consensus/polybft/polybft.go

```diff
@@ -846,17 +845,24 @@ func (p *Polybft) GetLatestChainConfig() (*chain.Params, error) {
 	return nil, nil
 }
 
-// GetBridgeProvider is an implementation of Consensus interface
-// Returns an instance of BridgeDataProvider
-func (p *Polybft) GetBridgeProvider() consensus.BridgeDataProvider {
-	return p.runtime
-}
-
 // FilterExtra is an implementation of Consensus interface
 func (p *Polybft) FilterExtra(extra []byte) ([]byte, error) {
 	return GetIbftExtraClean(extra)
 }
 
+func (p *Polybft) getChainIDs() []uint64 {
+	chainIDs := make([]uint64, len(p.genesisClientConfig.Bridge))
```

Comment:
```suggestion
	chainIDs := make([]uint64, 0, len(p.genesisClientConfig.Bridge))
```

And then remove the counter and just do append.